tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -103,6 +103,8 @@ const onlyOfficeCanvasEditorSelector = '#id_viewer_overlay'
 const onlyOfficeCanvasCursorSelector = '#id_target_cursor'
 const collaboraCanvasEditorSelector = '.leaflet-layer'
 const textEditorTextArea = '#text-editor-input'
+const filesContextMenuAction = '#oc-files-context-menu button.oc-files-actions-%s-trigger'
+const filescontextMenuPasteAction = '.oc-files-actions-copy-trigger'
```

Comment:
?

Reply:
Its actually for pasting the resource using dropdown-menu. I missed to add it in previous commit and added in later commit.

Reply:
I think, `filesContextMenuAction` should work, right?

Reply:
only if we use classname `tippy-box`. id for dropdown while pasting is different

Reply:
then maybe we can use for all
```js
const filesContextMenuAction = 'div[id^="context-menu-drop"] button.oc-files-actions-%s-trigger'
```